src/styles/editor-content/editor-content.ts

```diff
@@ -8,15 +8,55 @@ export const useEditorContentStyles = makeStyles({
     height: 72,
     display: "flex",
     alignItems: "center",
-    padding: `0px ${theme.spacing(3)}px`
+    padding: `0px ${theme.spacing(3)}px`,
+    "& h4": {
+      fontWeight: 600,
+      fontStyle: "italic"
+    },
+    "& h5": {
+      fontSize: "0.8rem"
+    }
```

Comment:
If there is anything you need to change for the heading globally, change it in the theme. Or local changes to typography can be applied using just style={{ ... }}.
and I am not sure is it good to use rem as the font unit. I normally just use fontSize: 16 for example. You can probably ask someone else about this one

Reply:
Those are actually "capsulated" styles and take place only on that specific component. I like to to keep styling in their own file. it gives better readability. Not sure about the advantages of rem but Material-Ui uses it, I'll ask Tuomas about this.